### PR TITLE
Jenkinsfile: add vagrant integration test

### PR DESCRIPTION
**Pull request by okeer**

### CODE REVIEW

**Comment by okeer** [comment]
retest this please

**Comment by okeer** [comment]
retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -9,5 +9,15 @@ pipeline {
         }
       }
     }
+    stage('Test with vagrant') {
+      steps {
+        sh '''#!/bin/bash
```
Comment (marvin4ward) [inline Jenkinsfile]: We need to start some script, or we will have here kind of a mess

Comment (okeer) [reply Jenkinsfile]: point taken

**Pull request merged**